src/test/java/com/fryrank/controller/MessageControllerTests.java

```diff
@@ -0,0 +1,24 @@
+package com.fryrank.controller;
+;
+
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.junit.Assert;
+
+import static com.fryrank.Constants.WELCOME_MESSAGE;
+
+public class MessageControllerTests {
+
+    @Autowired
+    private MessageController controller;
+
+    @Test
+    public void testGetWelcomeMessage() throws Exception {
+        MessageController controller = new MessageController();
+
+        String actualMessage = controller.getWelcomeMessage();
+        String expectedMessage = WELCOME_MESSAGE;
+
+        Assert.assertEquals(actualMessage, expectedMessage);
```

Comment:
Would recommend importing assertEquals only instead of importing entire Assert class

Reply:
Done in e560385f84676608e8101263f9702b6e5c9c5865.